### PR TITLE
automatic retrieval of healthcare capacity information for countries

### PR DESCRIPTION
**Pull request by cwhittaker1000**
get_healthcare_capacity function written to extract for a specified country:

- hosp and ICU beds where we have that data available for a particular country
- median hosp and ICU beds for the country's income strata (from World Bank) when we don't have estimates for that specific country

and have integrated the function into run_explicit_SEEIR :)

### CODE REVIEW

File: data-raw/hospital_bed_capacity.csv [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+Cabo Verde,2016,Lower middle income,Sub-Saharan Africa,NA,2.218710128,2.218710128
+Cambodia,2007,Lower middle income,East Asia & Pacific,NA,1.708706085,1.708706085
+Cameroon,2016,Lower middle income,Sub-Saharan Africa,NA,1.354560419,1.354560419
+Central African Republic,2016,Low income,Sub-Saharan Africa,NA,0.644838173,0.644838173
+Chad,2016,Low income,Sub-Saharan Africa,NA,0.364916345,0.364916345
+Chile,2015,High income,Latin America & Caribbean,NA,3.506149199,3.506149199
+China,2016,Upper middle income,East Asia & Pacific,NA,6.186642526,6.186642526
+Colombia,2016,Upper middle income,Latin America & Caribbean,NA,1.821092731,1.821092731
+Comoros,2014,Lower middle income,Sub-Saharan Africa,NA,0.84988754,0.84988754
+"Congo, Dem. Rep.",2015,Low income,Sub-Saharan Africa,NA,0.788912917,0.788912917
```
Comment (OJWatson) [inline data-raw/hospital_bed_capacity.csv:33]: These have " in front of them - Democratic Republic of the Congo is fine. Better to not abbreviate a country as it's harder to gues what the abbreviation is than typing the full name

File: data-raw/WPP_demog_matrix.csv [hunk 2]
```diff
@@ -1,202 +1,202 @@
...
-675,"Estimates","Zambia",4223.906936,NA,"LMIC",1,NA,894,"Country/Area",910,2020,2946.454,2717.088,2428.598,2121.583,1746.939,1433.444,1193.475,991.719,808.231,605.658,433.597,327.662,237.554,166.281,112.763,67.202,32.916,10.863,1.803,0.121,0.005,"ZMB","Sub-Saharan Africa","Lower middle income",4223.906936,15
-690,"Estimates","Zimbabwe",3029.793005,NA,"LMIC",1,NA,716,"Country/Area",910,2020,2097.448,2210.712,1921.06,1638.895,1377.829,1105.162,994.874,872.602,737.527,520.295,389.42,308.669,240.442,184.973,114.658,85.7,43.888,15.46,3.027,0.267,0.019,"ZWE","Sub-Saharan Africa","Lower middle income",3029.793005,15
+Index,Variant,"Region, subregion, country or area *",GDP2018,Notes,Income,Contact mat,Polymod,Country code,Type,Parent code,Reference date (as of 1 July),0-4,05-Sep,Oct-14,15-19,20-24,25-29,30-34,35-39,40-44,45-49,50-54,55-59,60-64,65-69,70-74,75-79,80-84,85-89,90-94,95-99,100+,country_code,region,income_group,GDP,Matrix
```
[the remark this answers is not in the document]
Comment (cwhittaker1000) [reply data-raw/WPP_demog_matrix.csv]: Have now sorted!

File: data-raw/hospital_bed_capacity.csv (hunk 1 above)
Comment (cwhittaker1000) [reply data-raw/hospital_bed_capacity.csv]: Have changed country names to get rid of the relevant "(Republic of)" random brackets stuff etc etc!

File: R/data.R [hunk 3]
```diff
@@ -13,3 +13,28 @@
 #' @format A data.frame
 # #' @source \url{http://www.diamondse.info/}
 "population"
+
+#' Country specific healthcare capacity
+#'
+#' Hospital and ICU beds per 1000 pop for various countries
+#'
+#' @format A data.frame
+# #' @source \url{http://www.diamondse.info/}
```
[the remark this answers is not in the document]
Comment (cwhittaker1000) [reply R/data.R]: Done!

**Comment by OJWatson** [comment]
LGTM - nice work with the tests.

**Pull request merged**